specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/putComputeOneSkuQuotaRequest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus",
+    "resourceName": "standardFSv2Family",
```

Comment:
>standardFSv2Family [](http://example.com/codeflow?start=21&length=18)

Should the resource name be one of predefined ones? If so, how does the user discover those?

Reply:
The resource name varies by RP.
The user should at first call Get ot list API and update the limit property.
We are building a framework to onboard all the RPs and the RPs will keep on adding the resources, so the user should follow the instructions at - https://docs.microsoft.com/en-us/rest/api/reserved-vm-instances/quotaapi